homeassistant/components/tami4/entity.py

```diff
@@ -0,0 +1,31 @@
+"""Base entity for Tami4Edge."""
+from __future__ import annotations
+
+from Tami4EdgeAPI import Tami4EdgeAPI
+
+from homeassistant.helpers.entity import DeviceInfo, Entity, EntityDescription
+
+from .const import DOMAIN
+
+
+class Tami4EdgeBaseEntity(Entity):
```

Comment:
New integrations/entities must support [translation of names and attributes](https://developers.home-assistant.io/blog/2023/03/27/entity_name_translations/).

Please adopt your code accordingly.